module/stats.py

```diff
@@ -18,21 +20,53 @@ def stats(update: Update, context: CallbackContext):
 def stats_gen(update: Update, context: CallbackContext, days):
     conn = sqlite3.connect('data/DMI_DB.db')
     chat_id = update.message.chat_id
-    query = ""
     text = ""
 
     if days == 0:
         text += "Record Globale:\n"
-        query = "SELECT Type, count(chat_id) FROM stat_list GROUP BY Type ORDER BY Type;"
+        query = "SELECT Type, n FROM (SELECT Type, COUNT(chat_id) as n FROM stat_list GROUP BY Type) ORDER BY n DESC;"
```

Comment:
You can't just use only one query inside than two nested for ordering?
probably because sqlite can't order by a count() variable?

Reply:
like 
```sql
SELECT Type, COUNT(chat_id) AS n FROM stat_list GROUP BY Type ORDER BY n DESC;
```